openlibrary/plugins/openlibrary/js/cover_add_manage.js

```diff
@@ -0,0 +1,58 @@
+export default function(){
+    // page may not be loaded via iframe
+    if (parent && parent.closeThrobber) {
+        parent.closeThrobber();
+    }
+    $('#form.addcover-form').submit(function(event) {
+        var file;
+        var url;
+        var coverid;
+        function val(selector) {
+            return $.trim($(selector).val());
+        }
+        function error(message) {
+            $('#errors').show().html(message);
+            event.preventDefault();
+        }
+        file = val('#coverFile');
```

Comment:
see above - let's declare these in a single line.
e.g
`var file = val('#coverFile');`